app/scripts/modules/core/src/pipeline/config/actions/delete/delete.module.js

```diff
@@ -28,8 +28,9 @@ module.exports = angular.module('spinnaker.core.pipeline.config.actions.delete',
               pipelineConfigService.savePipeline(pipeline);
             }
           });
-          application.pipelineConfigs.refresh();
-          $state.go('^.executions', null, {location: 'replace'});
+          application.pipelineConfigs.refresh().then(() => {
```

Comment:
Does it make more sense to just remove the `refresh` call here? It sounds like that call would like return stale data, if the delete doesn't clear the cache for a second and we call it immediately after performing the delete - and we're already removing the pipeline config above.

Reply:
Yea, that makes sense to me.  We only remove it if the call succeeds so that seems safe.